idseq_pipeline/commands/common.py

```diff
@@ -577,6 +578,20 @@ def download_reference_on_remote_with_version_any_source_type(
             key_path, remote_username, server_ip)
     return input_fasta_remote, version_number
 
+def big_version_change_from_last_run(pipeline_version, version_s3_path):
+    ''' Return True is there's a significant pipeline version chanage. i.e. 1.2.1 -> 1.3.0. False otherwise '''
+    try:
+        version_hash = json.loads(execute_command_with_output("aws s3 cp %s - " % version_s3_path))
+        for entry in version_hash:
+            if entry['name'] == 'idseq-pipeline':
+                prev_pipeline_version_sig = re.sub(r'\.\d+$', '', entry['version']);
+                pipeline_version_sig = re.sub(r'\.\d+$', '', pipeline_version)
+                return (prev_pipeline_version_sig != pipeline_version_sig)
+        return True # idseq-pipeline info is not
+    except:
+        # couldn't get the s3_version_file (no output most likely). Return True to indicate change from nothing
```

Comment:
Please add traceback.print_exc() before returning True here.   That way we'd see in the log what the exception was, just in case it's something unexpected.  (*pun*)